src/data/set/finite.lean

```diff
@@ -72,6 +72,15 @@ theorem exists_finite_iff_finset {p : set α → Prop} :
 ⟨λ ⟨s, hs, hps⟩, ⟨hs.to_finset, hs.coe_to_finset.symm ▸ hps⟩,
   λ ⟨s, hs⟩, ⟨↑s, finite_mem_finset s, hs⟩⟩
 
+lemma finite.fin_param {s : set α} (h : finite s) :
+  ∃ (n : ℕ) (f : fin n → α), injective f ∧ range f = s :=
+begin
+  classical,
+  obtain ⟨f⟩ := nonempty_of_trunc (fintype.equiv_fin (h.to_finset : set α)),
```

Comment:
```suggestion
  obtain ⟨f⟩ := (fintype.equiv_fin (h.to_finset : set α)).nonempty,
```
the lemma was renamed very recently

Reply:
I don't see this.

Reply:
You'll need to merge master. Your current version would fail CI when merged.

Reply:
Oh, I didn't understand you mean that recent.